requirements-development.txt

```diff
@@ -2,6 +2,6 @@
 pytest>=2.9.0,<3.0
 pytest-django
 pytest-factoryboy
-fake-factory
+Faker
```

Comment:
Thanks for picking up this package name change. This change has bit me on a few projects already.